packages/ia-topnav/src/styles/dropdown-menu.js

```diff
@@ -0,0 +1,105 @@
+import { css } from 'lit-element';
+
+export default css`
+  :host {
+    --topOffset: -1000px;
```

Comment:
What is this doing?

Reply:
Allowing for the quick change of the starting top offset. Since it's used in two spots, this gets rid of what Brenton typically points out to be a "magic constant".

Reply:
Oh I mean like what are you doing with it. Like what are you changing the top offset for?